lib/src/controls/better_player_material_controls.dart

```diff
@@ -62,6 +64,27 @@ class _BetterPlayerMaterialControlsState
           _cancelAndRestartTimer();
           _onPlayPause();
         },
+        onHorizontalDragStart: (_) {
```

Comment:
This code seems to be same as cupertino one. Can you create some shared class? DRY rule is broken here.